packer/build_deb_image.sh

```diff
@@ -292,14 +295,22 @@ export PACKER_LOG_PATH
 if $DRY_RUN ; then
   echo "DryRun: No need to grep errors on log"
 else
+  GREP_STATUS=0
   if [ "$TARGET" = "aws" ]; then
     grep "us-east-1:" $PACKER_LOG_PATH
+    GREP_STATUS=$?
   elif [ "$TARGET" = "gce" ]; then
       grep "A disk image was created" $PACKER_LOG_PATH
+      GREP_STATUS=$?
   elif [ "$TARGET" = "azure" ]; then
     grep "Builds finished. The artifacts of successful builds are:" $PACKER_LOG_PATH
+    GREP_STATUS=$?
+  else
+    echo "No Target is defined"
+    GREP_STATUS=1
   fi
-  if [ $? -ne 0 ] ; then
+  echo "GREP_STATUS: |$GREP_STATUS|"
```

Comment:
You don't need the `GREP_STATUS=1` definition in line 298
instead you can do:
```
if [ $? -ne 0 ] ; then
   echo "Say something"
   exit 1
else
   echo "say something else"
   exit 0
fi

Reply:
In general it is better to have a single exit point.
There are a few exit points in this script, but I don't want to add more.